commit 587c3feb407e486f71ab79cd87880533122ecb68
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-05-01T10:24:40+10:00 (5 months ago)
    
    MANTA-4242 Muppet fails to change haproxy config after a cueball/zk shuffle

diff --git a/package.json b/package.json
index c54fb9f..f5d072c 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "once": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
-        "zkstream": "0.11.3"
+        "zkstream": "0.11.4"
     },
     "devDependencies": {
         "nodeunit": "0.11.2",
